enterprise/views.py

```diff
@@ -523,6 +525,40 @@ def get_final_price(self, mode, request):
             )
         return mode['original_price']
 
+    @staticmethod
+    def post_order_to_ecommerce(request, sku):
+        """
+        Post an order for audit enrollment  for the user.
+        """
+        try:
+            api = ecommerce_api_client(request.user)
+            response_data = api.baskets.post({
+                'products': [{'sku': sku}],
+                'checkout': True,
+            })
+        except (HTTPError, Timeout, HttpClientError):
+            logger.error(
+                "Failed to post audit enrollment of user '{username}' in product '{sku}'.".format(
+                    sku=sku,
+                    username=request.user.username,
+                )
+            )
+            messages.error(
+                request,
+                _(
+                    "There was an error processing your request, please try again. "
```

Comment:
Since this will be a user facing message, it might be better to say something like "There was an error completing your enrollment in the course" for this sentence. @chris-mike thoughts? What error message should we show to the user if the enrollment call fails for some reason?